open3d_conversions/README.md

```diff
@@ -0,0 +1,76 @@
+# open3d_conversions
+
+This package provides functions that can convert pointclouds from ROS to Open3D and vice-versa.
+
+## Dependencies
+
+* Eigen3
+* Open3D
+
+## System Requirements
+
+* Ubuntu 18.04+: GCC 5+
+
+## Installation
+
+### Open3D
+
```

Comment:
This should point at a website / readme from Open3D itself for installing open3D in case this changes or becomes apt-get installable

Reply:
True, the only important part that should remain from this (at least until it becomes apt-get installable) is that the flags mentioned here should be on.

Reply:
I think that's on the Open3D instructions though - we don't want to maintain this in case things  change or its made available in apt-get. I filed a ticket for that for C++ bindings to use this work (python already released)

Reply:
Very well (f1905e6)